02-microarray/pathway-analysis_microarray_02_gsea.Rmd

```diff
@@ -204,36 +220,33 @@ Let's take a look at what these results from the differential expression analysi
 dge_df
 ```
 
-## Getting familiar with `clusterProfiler`'s options
+## Getting familiar with MSigDB gene sets available via `msigdbr`
+
+The Molecular Signatures Database (MSigDB) is a resource that contains annotated gene sets that can be used for pathway or gene set analyses [@Subramanian2005]. 
+We can use the `msigdbr` package to access these gene sets in a format compatible with the package we'll use for analysis, `clusterProfiler` [@Igor2020; @Yu2012].
 
-Let's take a look at what organisms the package supports.
+The gene sets available directly from MSigDB are applicable to human studies.
+`msigdbr` also supports commonly studied model organisms.
+
+Let's take a look at what organisms the package supports with `msigdbr_species()`.
 
 ```{r}
 msigdbr_species()
 ```
 
-MSigDB contains 8 different gene set collections [@Subramanian2005].
+MSigDB contains [8 different gene set collections](https://www.gsea-msigdb.org/gsea/msigdb/collections.jsp) [@Subramanian2005] that are distinguished by how they are derived (e.g., computationally mined, curated).
```

Comment:
We should probably use the preferred citation here: (Liberzon et al. 2011)
```
@article{Liberzon2011,
  doi = {10.1093/bioinformatics/btr260},
  url = {https://doi.org/10.1093/bioinformatics/btr260},
  year = {2011},
  month = may,
  publisher = {Oxford University Press ({OUP})},
  volume = {27},
  number = {12},
  pages = {1739--1740},
  author = {Arthur Liberzon and Aravind Subramanian and Reid Pinchback and Helga Thorvaldsdóttir and Pablo Tamayo and Jill P Mesirov},
  title = {Molecular signatures database ({MSigDB}) 3.0},
  journal = {Bioinformatics}
}
```

Reply:
Ah, you mean in addition to `@Subramanian2005` correct?

> To cite your use of the Molecular Signatures Database (MSigDB), a joint project of UC San Diego and Broad Institute, please reference Subramanian, Tamayo, et al. (2005, PNAS 102, 15545-15550) and one or more of the following as appropriate: Liberzon, et al. (2011, Bionformatics), Liberzon, et al. (2015, Cell Systems), and also the source for the gene set as listed on the gene set page. 

Where the 2015 citation is hallmarks, specifically.

Reply:
Yes, in addition...

Reply:
👍  I propagated this throughout the pathway analysis notebooks I've gone through so far.